packages/generational-box/README.md

```diff
@@ -31,4 +31,4 @@ let store = Store::default();
 
 ## How it works
 
-Internally 
\ No newline at end of file
+Internally <!-- Cool -->
```

Comment:
We probably shouldn't include <!-- Cool --> here in the official version of the readme

Reply:
Haha yeah. Just found "How it works: Internally" funny.

Reply:
Yeah... I need to finish writing that